src/pages/ContactList/ContactList.jsx

```diff
@@ -5,7 +5,7 @@ const ContactList = () => {
 
     return (
         <div className="contactList">
-            <ContactContent />
+            <ContactContent numContacts={10} />
```

Comment:
hardcoded?

Reply:
Hi Kirsty, this is so that it does not all show contacts on the contact list page. Show a certain amount